src/Nowruz/pages/contribute/index.tsx

```diff
@@ -0,0 +1,183 @@
+              theme="success"
+              iconName="check-circle"
+              title="Congratulations on reaching 10,000 impact points!"
+              subtitle="You're now invited to join our Socious Contributor community and make an even bigger impact on our platform. Click to join now!"
+              colOrderMobileView
+            />
+          ) : (
+            <AlertMessage
+              theme="warning"
+              iconName="alert-circle"
+              title="Keep Going to unlock the Socious Contributor program!"
+              subtitle="You're making great progress! Reach 10,000 impact points to join our exclusive Contributor program and make an even bigger impact on the Socious platform."
+              colOrderMobileView
+            />
+          )}
+          <div className={css.supprtingText}>
+            As a Socious Contributor, you&apos;ll have the unique opportunity to play an active role in various aspects
+            of our platform, including dispute resolution, content moderation, and profile verification. By joining our
+            contributor pool, you&apos;ll work alongside like-minded individuals to ensure the integrity, fairness, and
+            safety of the Socious community.
+          </div>
+
+          <Button variant="text" color="primary" customStyle="!p-0 flex flex-row gap-2">
+            <div className={`${css.bold} text-Brand-700 underline`}>
+              Learn more about the Socious Contributor Program
+            </div>
+            <Icon name="arrow-right" fontSize={20} className="text-Brand-700" />
+          </Button>
+          <div className="text-xl font-semibold leading-[30px] text-Gray-light-mode-900">Why become a contributor?</div>
+          <div className="flex flex-col gap-5 pl-4">
+            {renderCheckItems(
+              'Shape the platform:',
+              ' Your contributions directly influence the user experience and help maintain a trustworthy and engaging environment for all Socious members.',
+            )}
+            {renderCheckItems(
+              'Develop valuable skills:',
+              ' Participating as a contributor allows you to develop your problem-solving and decision-making abilities.',
+            )}
+            {renderCheckItems(
+              'Earn rewards:',
+              ' To recognize your time and effort, contributors are eligible for rewards such as impact points, exclusive access to features, or other incentives based on their level of involvement.',
+            )}
+          </div>
+          {
+            // TODO: onClick open join modal
+            eligible && (
+              <Button variant="contained" color="primary" customStyle=" mx-auto">
+                Join now
+              </Button>
+            )
+          }
+          <div className="text-xl font-semibold leading-[30px] text-Gray-light-mode-900">How to get started </div>
+          <div className="flex  flex-col">
+            {renderSteps(
+              0,
```

Comment:
for this `renderSteps` also you can use array mapping, it is more readable and separates constant data from your JSX